indico/modules/events/papers/templates/_contributions.html

```diff
@@ -48,6 +51,7 @@ <h3 class="icon-file">{% trans %}Paper{% endtrans %}</h3>
 {% endmacro %}
 
 
+
```

Comment:
We usually only have 2 blank lines between macros (so I'd only go for 3 if it really improves readability)